### ISSUE TITLE
New yodaConditions rule broken

### ISSUE THREAD

**Issue by vincentisambart** [nicklockwood/SwiftFormat#383]
Formatting the following code with 0.40.1:
```swift
import UIKit

class ABCD: UIViewController {
    func foo(vc: UIViewController) {
        if navigationController.viewControllers[0] == vc {
            print("equal")
        }
    }
}
```
gives me the following incorrect Swift code:
```swift
import UIKit

class ABCD: UIViewController {
    func foo(vc: UIViewController) {
        if navigationController.viewControllersvc == [0] {
            print("equal")
        }
    }
}
```
It doesn't happen anymore if you disabling the `yodaConditions` rule or if you go back to 0.39.5 so it's probably due to the changes to `yodaConditions` in 0.40.0.

**Comment by nicklockwood** [comment]
@vincentisambart thanks for reporting. I've found the problem and will push a fix asap.

**Comment by nicklockwood** [comment]
@vincentisambart I've pushed a fix for this bug to the [develop](https://github.com/nicklockwood/SwiftFormat/tree/develop) branch if you'd like to try it (you'll need to build the command-line app from source, as I've not updated the binary).

**Comment by vincentisambart** [comment]
@nicklockwood Thanks! I just try the latest develop branch and it does indeed seem to fix the problem 👍

**Comment by nicklockwood** [comment]
@vincentisambart great! The new version should be ready shortly.

**Comment by nicklockwood** [comment]
@vincentisambart fixed in 0.40.2.
